utils/coreutils/techutils.go

```diff
@@ -29,13 +29,30 @@ const (
 	Go     Technology = "go"
 	Pip    Technology = "pip"
 	Pipenv Technology = "pipenv"
+	Pypi   Technology = "pypi"
 	Poetry Technology = "poetry"
 	Nuget  Technology = "nuget"
 	Dotnet Technology = "dotnet"
 	Docker Technology = "docker"
 )
 
-const Pypi = "pypi"
+func TechnologyToLanguage(technology Technology) string {
+	languageMap := map[Technology]string{
+		Npm:    "JavaScript",
+		Pip:    "Python",
+		Poetry: "Python",
+		Pipenv: "Python",
+		Pypi:   "Python",
+		Go:     "Go",
+		Maven:  "Java",
+		Gradle: "Java",
+		Nuget:  "C#",
+		Dotnet: "C#",
+		Yarn:   "JavaScript",
+		Pnpm:   "JavaScript",
+	}
+	return languageMap[technology]
+}
```

Comment:
```suggestion
type CodeLanguage string

const (
    JavaScript CodeLanguage = "javascript"
    Python     CodeLanguage = "python"
    GoLang     CodeLanguage = "go"
    Java       CodeLanguage = "java"
    CSharp     CodeLanguage = "C#"

)
```
and add `Language` attribute to the `TechData` objects